### DISCUSSION TITLE
Right Recognition Wrong Raw Value

### DISCUSSION THREAD

**Discussion by T1gg1** [jomjol/AI-on-the-edge-device#2895] [General]
### Issue

![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/77970513/c9a0defd-77ff-4afb-982c-ef0b073c1687)
Hey
as you see my esp recognized the right numbers but the raw value is wrong. Know anyone why?

BR

**Comment by SybexX** [comment]
why should the raw value be wrong???
The first analog digit is recognized as 6.1, but this is reduced to 5 by the second analog digit, 
since for 68 the first analog digit should almost be at 7

**Comment by SybexX** [comment]
The values displayed are not the real ones because they have been post-processed, therefore they can only be used as a rough guide.
As far as I could tell from the code, the displayed values are only rounded, against what the raw value is calculated with great effort.

**Comment by Slider0007** [comment]
For me also, value looks correct.
The post processing algo corrected the 6.1 -> 5 because the lower number (8.3) does not have a zero crossing yet.

The single ROI values a RAW values from model.
The value on top is the resulting value after post processing.

Info about rounding:
https://github.com/jomjol/AI-on-the-edge-device/issues/2830#issuecomment-1914852290

**Comment by caco3** [comment]
The recognition page can only be used to validate the individual recognition of each digit/pointer.
The overall value gets evaluated in a more complex algorithm.
You don't state that it is wrong there, so I am not sure what your issue is!
